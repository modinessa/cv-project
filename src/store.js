import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
