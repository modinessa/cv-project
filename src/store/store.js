import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});