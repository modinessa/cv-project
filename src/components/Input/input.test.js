import { Provider } from "react-redux";
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Input } from "./index";

const TEST_NAME = 'name';
const TEST_TITLE = 'Title';
const TEST_ERRORS = {[TEST_NAME]: {message: 'test error'}};

describe('Input', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Input
					register={() => {}}
					name={TEST_NAME}
					title={TEST_TITLE}
					errors={{TEST_ERRORS}}
					/>
			</Provider>
		)).toMatchSnapshot();
	})
})