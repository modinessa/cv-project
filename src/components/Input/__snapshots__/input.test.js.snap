// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders component 1`] = `
Array [
  <div
    className="data-row"
  >
    <label
      className="title-column"
      htmlFor="name"
    >
      Title:
    </label>
    <input
      className="data-column skill-input "
      placeholder="Enter title"
    />
  </div>,
  <div
    className="error-message"
  />,
]
`;
